src/sentry/plugins/bases/issue2.py

```diff
@@ -111,6 +59,36 @@ def get_group_title(self, request, group, event):
     def is_configured(self, request, project, **kwargs):
         raise NotImplementedError
 
+    def get_group_urls(self):
+        _urls = []
+        for action in self.allowed_actions:
+            view_method_name = 'view_%s' % action
+            _urls.append(
+                url(r'^%s/' % action,
+                    PluginGroupEndpoint.as_view(
+                        view_method_name=view_method_name,
```

Comment:
I will probably explore if we can tweak these and do ``view=getattr(self, view_method_name)``